roles/openshift_cluster_autoscaler/templates/cluster-autoscaler.j2

```diff
@@ -20,8 +20,9 @@ spec:
     spec:
       containers:
       - args:
+        - /bin/cluster-autoscaler
         - --alsologtostderr
-        - --v={{ openshift_cluster_autoscaler_version }}
+        - --v={{ openshift_cluster_autoscaler_loglevel }}
         - --skip-nodes-with-local-storage={{ openshift_cluster_autoscaler_skip_nodes_local_storage }}
         - --leader-elect-resource-lock=configmaps
         - --namespace={{ openshift_cluster_autoscaler_namespace }}
```

Comment:
So we still need a way to change the `--scale-down-delay-after-failure`, `--scale-down-unneeded-time` and `--scale-down-delay-after-add` before we deploy the autoscaler so we get faster response. What about this:
```yaml
{% if openshift_cluster_autoscaler_scale_down_delay_after_failure is defined %}
        - --scale-down-delay-after-failure={{ openshift_cluster_autoscaler_scale_down_delay_after_failure }}
{% endif %}
{% if openshift_cluster_autoscaler_scale_down_unneeded_timeout is defined %}
        - --scale-down-unneeded-time={{ openshift_cluster_autoscaler_scale_down_unneeded_timeout }}
{% endif %}
{% if openshift_cluster_autoscaler_scale_down_delay_after_add is defined %}
        - --scale-down-delay-after-add={{ openshift_cluster_autoscaler_scale_down_delay_after_add }}
{% endif %}
```

?

Reply:
> What about this:

Done.